src/gui/PasswordEdit.cpp

```diff
@@ -105,3 +108,24 @@ void PasswordEdit::autocompletePassword(const QString& password)
         setText(password);
     }
 }
+
+void PasswordEdit::secureInputEntry(bool b)
```

Comment:
The build is failing on Linux (and presumably Windows) because this `bool b` isn't used except on macOS. I've only got the `#ifdef` here, as I was thinking that if a similar feature is implemented on Windows/Linux, then it can be implemented conveniently here (though there's probably low chance the API looks the same?).

Reply:
Solved with `Q_UNUSED`